schema.sql

```diff
@@ -7,6 +7,7 @@ CREATE TABLE IF NOT EXISTS `accounts` (
   `lastday` int(10) unsigned NOT NULL DEFAULT '0',
   `email` varchar(255) NOT NULL DEFAULT '',
   `creation` int(11) NOT NULL DEFAULT '0',
+	`secret` char(16) DEFAULT NULL,
```

Comment:
* There's unusual whitespace
* The column should be added after `password` to be consistent with https://github.com/otland/forgottenserver/blob/1.2/data/migrations/18.lua